bin/sqlToLog

```diff
@@ -124,16 +128,18 @@ ORDER BY visit %s
                     continue
                 if re.search(r"^(COMMENT\s+|END$)", card):
                     continue
-                
+
                 mat = re.search(r"^([^\s]+)\s*=\s*([^\s]+)", card)
                 if mat:
                     k, v = mat.groups()
                     if v in ("T", "F"):
                         v = (v == "T")
                     else:
-                        try: v = int(v)
+                        try:
+                            v = int(v)
                         except:
-                            try: v = float(v)
+                            try:
+                                v = float(v)
                             except:
                                 v = v[1:-1].rstrip()
 
```

Comment:
Whats going on here is Lupton regex magic that I totally don't want to touch. I'd bet this whole thing could be reduced to about 3 lines if it was done smarter, but: not tested and it's too much magic, and I have no idea what it's for.